plugins/chat/lib/chat_channel_archive_service.rb

```diff
@@ -48,6 +48,21 @@ def self.retry_archive_process(chat_channel:)
     chat_channel.chat_channel_archive
   end
 
+  def self.validate_topic_params(guardian, topic_params)
```

Comment:
It feels weird that you have to initialize a TopicCreator just to validate topic params. Is it worth refactoring this maybe?

Reply:
What would you suggest instead?

Reply:
Extracting validation logic to a new class TopicValidator or something like that, if possible of course.

Reply:
`TopicCreator` is the thing that is best suited to validate these params, I don't know why we would use a different class.

Reply:
`TopicCreator` does a lot of things already. What I was hinting at is extracting validation logic to `TopicValidator` and letting `TopicCreator` use that. Then, when we need validation only we can just use `TopicValidator`. We have a similar pattern for posts with `PostCreator` and `PostValidator`. 

Anyway, this is not strictly related to this commit, but a nice thing to have.

Reply:
Yep agreed it would make sense to have a counter part to `PostValidator`, arguably out of the scope of the PR yes.

Reply:
Yeah this is not in scope at all for this PR, feel free to open an internal TODO for the refactor @nbianca